tests/rt.conf

```diff
@@ -250,3 +250,10 @@ RUN     | atmaero_control_p8_rad
 
 COMPILE | -DAPP=ATMAQ -DCCPP_SUITES=FV3_GFS_v15p2                                                                               |                                           | fv3 |
 RUN     | regional_atmaq                                                                                                        |                                           | fv3 |
+
+###################################################################################################################################################################################
+# ATM-CATCHEM tests                                                                                                                                                               #
+###################################################################################################################################################################################
+
+COMPILE | -DAPP=ATMAERO -DCCPP_SUITES=FV3_GFS_v17_p8_catchem -D32BIT=ON                                                         | - wcoss2.intel                            | fv3 |
```

Comment:
this is my fault.... we should not include AERO here when compiling the APP 

please change `-DAPP=ATMAERO` to `-DAPP=ATM`

Reply:
@bbakernoaa done. Thanks!